src/main/java/com/sobytylnik/quiz/QuizFactory.java

```diff
@@ -0,0 +1,31 @@
+package com.sobytylnik.quiz;
+
+public class QuizFactory {
+
+    public static Quiz createHobbieQuiz(int id, String text, String description) {
+        return new Quiz(QuizType.HOBBIES, id, text, description);
```

Comment:
I supposed the factory methods should create a quiz with concrete questions and possible answers.
For example:
```
public static Quiz createHobbieQuiz() {
    Quiz quiz = new Quiz(QuizType.HOBBIES, 1, "Hibbies Quiz", "This quiz tests you hobbie preferences");

    Question question1 = new Question(1, 1, "Do you like sport?", false);
    question1.addOption(new Option(1, 1, "Yes"));
    question1.addOption(new Option(2, 2, "No"));
    quiz.addQuestion(question1);

    Question question2 = new Question(2, 2, "What genre of music do you like?", true);
    question2.addOption(new Option(3, 1, "Rock"));
    question2.addOption(new Option(4, 2, "Pop"));
    question2.addOption(new Option(5, 3, "Metal"));
    question2.addOption(new Option(6, 4, "Classic"));
    question2.addOption(new Option(7, 5, "Jazz"));
    quiz.addQuestion(question2);

    return quiz;
}
```